tools/mipgen/src/main.cpp

```diff
@@ -217,6 +231,34 @@ int main(int argc, char* argv[]) {
     vector<LinearImage> miplevels(count);
     generateMipmaps(sourceImage, g_filter, miplevels.data(), count);
 
+    if (g_ktxContainer) {
```

Comment:
It'd be awesome to add support for KTX to roughness-prefilter :)

Reply:
Will do :)